tegra/tegra.c

```diff
@@ -206,7 +206,7 @@ int drm_tegra_bo_unref(struct drm_tegra_bo *bo)
 		return -EINVAL;
 
 	if (!atomic_dec_and_test(&bo->ref))
-		return 0;
+		return atomic_read(&bo->ref);
```

Comment:
@digetx: Good point. I guess libdrm is always compiled without defining NDEBUG, so asserts are always enabled. Strange, but it is what it is.